source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
+A subscription allows you to get notified of updates on a datasets' data. We strongly recommend that you read the [dataset concept](#dataset) and [dataset endpoints](#dataset6) sections before proceeding.
+
+In the following sections, we will cover how you can interact and manage subscriptions in the RW API. We will see how you can customize subscriptions so that we only get notified for changes in a dataset's data for specific geographic regions. You will learn how to use them to get notified via email or calls to a URL you provide. We will also dive into subscription lifecycle, and understand how we can confirm subscriptions, resend confirmation emails and how to unsubscribe and stop receiving notifications.
+
+However, we will start by understanding how we can prepare datasets to support subscriptions.
 
 ## Subscribable datasets
 
-In order to create a subscription, a dataset must be prepared to accept them. This is achieved by declaring some queries in the dataset json object sent to the `/v1/dataset` endpoint under the `subscribable`property.
+> Example of a subscribable dataset
 
 ```json
 {
-	"name": "dataset name",
-	"application": ["app"],
-	"provider": "dataset provider",
-	"other": "dataset_fields",
-    "subscribable": {
-        "test_subscription": {
-            "dataQuery": "SELECT * FROM dataset-name  WHERE \"reported_date\" >= '{{begin}}' AND \"reported_date\" <= '{{end}}' AND \"number_dead\" > 0 ORDER BY reported_date DESC LIMIT 10",
-            "subscriptionQuery": "SELECT COUNT(*) FROM dataset-name WHERE \"reported_date\" >= '{begin}' AND \"reported_date\" <= '{end}' AND \"number\" > 0"
-        }
+  "name": "Example dataset",
+  "application": ["app"],
+  "provider": "carto",
+  ...
+  "subscribable": {
+    "test": {
+      "dataQuery": "SELECT * FROM dataset-name  WHERE 'reported_date' >= '{{begin}}' AND 'reported_date' <= '{{end}}' AND 'number_dead' > 0 ORDER BY reported_date DESC LIMIT 10",
+      "subscriptionQuery": "SELECT COUNT(*) FROM dataset-name WHERE 'reported_date' >= '{begin}' AND 'reported_date' <= '{end}' AND 'number' > 0"
     }
+  }
 }
 ```
 
-Inside the `subscribable` object, one (or several) sub-objects can be declared. In the provided example, the `test_subscription` is provided, with both a `dataQuery` and a `subscriptionQuery`. The former will be evaluated as the subscription's content, while the latter will be evaluated to check if a subscription has changed since the last update. Both queries can contain two special keywords: {begin} and {end}. These will be substituted with  ansi-formatted datetimes --with the datetime in which the subscription was last evaluated, and the datetime at the time of evaluation, respectively.
+Before we go into the details of managing subscriptions, it's important to understand that, while you can create a subscription for *any* dataset, some conditions must be met by the dataset for its corresponding subscriptions to be functional and actually work as described here.
```

Comment:
Assuming I understand correctly, we should create a backlog item to return error responses if a dataset is not configured for subscriptions. @ikas can you confirm that this would make sense?

Reply:
@ethanroday yes, we should!